tasks/create-pyxis-image/tests/test-create-pyxis-image-one-containerimage-multi-arch.yaml

```diff
@@ -71,58 +71,58 @@ spec:
           - name: check-result
             image: quay.io/konflux-ci/release-service-utils:65d8db844c008e7736ec8dff307868525279484d
             script: |
-              #!/usr/bin/env sh
+              #!/usr/bin/env bash
               set -eux
 
-              if [ $(cat $(workspaces.data.path)/mock_create_container_image.txt | wc -l) != 2 ]; then
+              if [ "$(wc -l < "$(workspaces.data.path)"/mock_create_container_image.txt)" != 2 ]; then
                 echo Error: create_container_image was expected to be called 2 time. Actual calls:
-                cat $(workspaces.data.path)/mock_create_container_image.txt
+                cat "$(workspaces.data.path)/mock_create_container_image.txt"
                 exit 1
               fi
 
-              if [ -f $(workspaces.data.path)/mock_cleanup_tags.txt ]; then
+              if [ -f "$(workspaces.data.path)"/mock_cleanup_tags.txt ]; then
                 echo Error: cleanup_tags was not expected to be called. Actual calls:
-                cat $(workspaces.data.path)/mock_cleanup_tags.txt
+                cat "$(workspaces.data.path)/mock_cleanup_tags.txt"
                 exit 1
               fi
 
-              if ! grep -- "--tags testtag" < $(workspaces.data.path)/mock_create_container_image.txt 3> /dev/null
+              if ! grep -- "--tags testtag" < "$(workspaces.data.path)/mock_create_container_image.txt" 3> /dev/null
```

Comment:
No idea what the `3>` is for. But I googled and apparently you can create your own descriptors with any number higher than 2. But you would need to start them first. 

```
exec 3>output.txt  # Redirect file descriptor 3 to "output.txt"
echo "This will go to the file output.txt" >&3  # Use file descriptor 3
exec 3>&-  # Close file descriptor 3
```

So this looks like a typo. Can you change it to `2>` please? Also in another place.

Reply:
Agree. It looks like a typo before and I didn't notice it. Will update. Thanks

Reply:
Thanks